content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/snips.sh

```diff
@@ -190,9 +190,9 @@ stream {
 
   # tcp forward proxy by SNI
   server {
-    resolver 8.8.8.8 ipv6=off;
+    resolver 8.8.8.8 ipv6=off; #if using in kubernetes it may help to use the kube dns resolver (kube-dns.kube-system.svc.cluster.local)
     listen       127.0.0.1:18443;
-    proxy_pass   \$ssl_preread_server_name:443;
```

Comment:
I think the `\` is needed to escape the `$` char because of the way the `cat` command is used to put the contents into the file `sni-proxy.conf`. That bash command will remove the `\`.

Btw, you should never modify `snips.sh` files directly. Instead, go to the corresponding .md file (`wildcard-egress-hosts/index.md` in this case), make the change there, and then run `make gen` to regen the snips file. The way that snips are generated from md files is described here: https://github.com/istio/istio.io/blob/master/tests/README.md